src/utils/hearing.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+
+
+export function getMainSection(hearing) {
+  return _.find(hearing.sections, (section) => section.type === "main");
+}
+
+/*
+* Return URL to hearing view. Accepts optional fullscreen parameter
+* to force fullscreen query parameter.
+ */
+export function getHearingURL(hearing, {fullscreen} = {}) {
+  const url = `/hearing/${hearing.id}`;
```

Comment:
Change it now to use `hearing.slug` instead of `id`. Also refactored `OverviewMap` so that it uses this same function. I think this should now solve #247 altogether.